movie_list.md

```diff
@@ -58,3 +58,42 @@
 * Looper
 * Braveheart
 * Gladiator
+
+Honorable mentions:
```

Comment:
Either make this into a H2 (##) and add one to the top, or delete these two rows.